src/components/VerticalActionsList.js

```diff
@@ -58,12 +60,18 @@ function VerticalActionsList({ goalAddress, goal, onArchiveClick, updateGoal })
         <span>squirrels</span>
       </div>
       {viewsOpen.squirrels && <PeoplePicker onClose={() => setViews({ ...defaultViews })} />}
-      {/* Hierarchies */}
+      {/* hierarchy */}
       <div className='action_list_item' key='hierarchies' onClick={() => setViews({ ...defaultViews, hierarchy: !viewsOpen.hierarchy })}>
         <Icon name='hierarchy_white.svg' />
         <span>hierarchy</span>
       </div>
       {viewsOpen.hierarchy && <HierarchyPicker onClose={() => setViews({ ...defaultViews })} selectedHierarchy={goal.hierarchy} hierarchyClicked={updateGoalHierarchy} />}
+      {/* priority */}
+      <div className='action_list_item' key='priority' onClick={() => setViews({ ...defaultViews, priority: !viewsOpen.priority })}>
+        <Icon name='priority_white.svg' />
+        <span>priority</span>
+      </div>
+      {viewsOpen.priority && <Priority onClose={() => setViews({ ...defaultViews })} selectedPriority={goal.priority} />}
```

Comment:
goal.priority doesn't exist yet, but that's ok, that's just placeholder for future implementation of the backend stuff and 'wiring it up'

Reply:
yeah, I know :)